jobs/base_schedulers.go

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package jobs
+
+import (
+	"time"
+
+	"github.com/mattermost/mattermost-server/v6/model"
+	"github.com/mattermost/mattermost-server/v6/shared/mlog"
+)
+
+type PeriodicScheduler struct {
+	jobs        *JobServer
+	minutes     int
+	jobType     string
+	enabledFunc func(cfg *model.Config) bool
+}
+
+func NewPeridicScheduler(jobs *JobServer, jobType string, minutes int, enabledFunc func(cfg *model.Config) bool) *PeriodicScheduler {
```

Comment:
Typo:  should be `NewPeriodicScheduler` here and where called.

Reply:
:man_facepalming: